client/components/Dashboard/icons/ImportExportIcon.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { Component, PropTypes } from 'react';
```

Comment:
this should be no longer added to client/components/Dashboard/icons/ImportExportIcon.jsx, but client/components/Icons/ImportExportIcon.jsx... please rebase with dev branch..